### PR TITLE
display menus based on settings

### PR DESCRIPTION
**Pull request by bucko13**
Interacts with this update in price-settings: https://github.com/bpanel-org/price-settings/pull/1

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -78,45 +80,50 @@ export const decorateFooter = (Footer, { React, PropTypes }) => {
...
         refreshPrice
       } = this.props;
 
       const Ticker = () => (
-        <div className={'col'} style={{height: '100%', lineHeight: '200%'}}>
+        <div className={'col'} style={{ height: '100%', lineHeight: '200%' }}>
           <Text>
-            {price.crypto +
-              ' ' +
+            {'1 ' + price.crypto +
```
Comment (pinheadmz) [inline lib/index.js]: Do you think it's not obvious to crypto users that the displayed fiat amount equals `1` of the crypto? :-D

Comment (bucko13) [reply lib/index.js]: I don't see it so much as as a crypto thing as just a exchange rate convention. Just reads a little clearer to me, like when you're at an airport exchanging money.

Comment (pinheadmz) [reply lib/index.js]: Ah hmm ok I'm just not in love with how it looks, takes up a little bit extra space for what I see as redundant. I don't see the `1 BTC =` on any other Bitcoin website

Comment (bucko13) [reply lib/index.js]: That's fair. At a certain level it is just a personal preference thing I guess and this is your widget :).

For context though, in "normal" exchanges this is what I've seen (and how about that headline!) from https://money.cnn.com/data/currencies/:
screen shot 2019-01-14 at 7 59 36 pm

I find it helpful when you're using a UI that can be a number of different combos. But again, personal preference! Happy to change if you prefer.

File: lib/index.js [hunk 2]
```diff
@@ -78,45 +80,50 @@ export const decorateFooter = (Footer, { React, PropTypes }) => {
...
-              refreshPrice();
-            }}
-          >
-            {price.availableFeeds.map(val => (
-              <option value={val} key={val}>
-                {val}
-              </option>
-            ))}
-          </select>
+          {!hideMenus && (
```
Comment (pinheadmz) [inline lib/index.js]: The only issue I see with this, is that not all currencies have a symbol in front of the value. So this menu is the only indicator what currency you are on. Might be fine without? Or maybe just leave the text,  but not a menu?

Comment (bucko13) [reply lib/index.js]: I guess this is what prompted me for the update above. I could make the arrangement/styles above conditional on if there is a menu or not.

**Comment by bucko13** [comment]
Should've included this in the description. Sorry.
No menus:
screen shot 2019-01-15 at 10 09 26 am
with menus:
screen shot 2019-01-15 at 10 09 35 am

Different currency pair:
screen shot 2019-01-15 at 10 10 48 am

screen shot 2019-01-15 at 10 10 37 am

**Comment by bucko13** [comment]
Alternatively could be displayed like this:
price

This style is a little closer to coinbase's exchange:
screen shot 2019-01-15 at 10 37 07 am

**Comment by pinheadmz** [comment]
Ah yeah I like this better!
Try also with `blockchain_info`/`ISK` (Icelandic Krona?) There is no unicode symbol in the widget (I suppose I could add... I think its just `kr`) But see if you like how that looks when there is no `$` etc in front of the number.

This layout is making me wonder if we could replace the fiat symbol in the pair WITH the menu? So it'd look like:

Menus on:
`[Coinbase]` BTC/`[USD]` $3,800

Menus off:
BTC/USD $3,800

**Comment by tynes** [comment]
This is looking really cool!

**Comment by bucko13** [comment]
Without a symbol I think it still looks good, just a little less clear:
screen shot 2019-01-15 at 5 30 14 pm

**Comment by bucko13** [comment]
As for using the menu in place of the fiat ticker when menus are on, I feel like it could work if we had a specially stylized menu but the html (or our own dropdown) feel a little too bulky. I can try it out though.

**Comment by bucko13** [comment]
It definitely does feel less redundant. Whadya think? 
screen shot 2019-01-15 at 5 51 54 pm

**Comment by pinheadmz** [comment]
YES! I dig it - nice jam!

**Comment by pinheadmz** [comment]
Really like how it looks with the menus hidden and the text still there - nice job!
👍

**Review by pinheadmz — APPROVED** [review]

**Pull request merged**